### PR TITLE
remove the people models

### PR DESCRIPTION
**Pull request by Pomax**
Part 1 of 2 for https://github.com/mozilla/foundation.mozilla.org/issues/4908 (part 2: https://github.com/mozilla/foundation.mozilla.org/pull/5724)

Removes the model and adds a migration to effect that.

### CODE REVIEW

File: network-api/networkapi/people/urls.py [hunk 1]
```diff
@@ -1,11 +1,2 @@
-from django.conf.urls import url
-
-from networkapi.people.views import (
-    PeopleListView,
-    PersonView,
-)
-
 urlpatterns = [
```
Comment (KalobTaulien) [inline network-api/networkapi/people/urls.py:1]: Is this file needed at all anymore if the urlpatterns are empty?

**Review by KalobTaulien — APPROVED** [review]
Looks good to me 👍  We could probably delete `network-api/networkapi/people/urls.py` since it's basically empty now. But that's non blocking for this PR.

File: network-api/networkapi/people/urls.py (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/people/urls.py:1]: it is not, but part 2 removes it too
